doc/api/https.md

```diff
@@ -46,7 +46,7 @@ added: v8.0.0
 
 See [`http.Server#keepAliveTimeout`][].
 
-## https.createServer(options[, requestListener])
+## https.createServer([options][, requestListener])
```

Comment:
@refack the test is in **test/parallel/test-https-immutable-options.js**, which tested in `server2`.